commons/all/src/main/java/org/infinispan/commons/api/CacheContainerAdmin.java

```diff
@@ -19,21 +22,25 @@ public interface CacheContainerAdmin<C extends CacheContainerAdmin, A extends Ba
     * @author Tristan Tarrant
     * @since 9.2
     */
+   @ProtoTypeId(ProtoStreamTypeIds.CACHE_CONTAINER_ADMIN_FLAG)
    enum AdminFlag {
       /**
        * If the operation affects configuration, make it permanent, which means it will survive restarts.
        * If the server cannot honor this flag an error will be returned
        * @deprecated configurations are now always permanent by default. Use the {@link #VOLATILE} flag to obtain the opposite behaviour.
        */
       @Deprecated(forRemoval=true, since = "11.0")
```

Comment:
Remove the deprecated value in this PR or a separated PR.

Reply:
I've created a separate PR for the feature branch: https://github.com/infinispan/infinispan/pull/13651